libs/dh/globalization/assets-localization/src/assets/i18n/da.json

```diff
@@ -507,8 +507,8 @@
       "processes": {
         "tabLabel": "Procesoverblik"
       },
-      "prices": {
-        "tabLabel": "Priser"
+      "charge-links": {
+        "tabLabel": "Pris tilknytninger"
```

Comment:
question: shouldn't it be "Priser"? If not, then it should be in one word: "Pristilknytninger"